ext/standard/basic_functions.stub.php

```diff
@@ -1747,7 +1750,10 @@ function array_intersect_key(array $array, array ...$arrays): array {}
  */
 function array_intersect_ukey(array $array, ...$rest): array {}
```

Comment:
Why was this skipped?

Reply:
Can callable be CTE? (first class callable?)

also one question - is the stub correct? - https://www.php.net/manual/en/function.array-intersect-ukey.php the callback is not present

Reply:
Sorry, of course not. The signature of this function is weird so I didn't realize this takes a callable.

Reply:
> also one question - is the stub correct? - https://www.php.net/manual/en/function.array-intersect-ukey.php the callback is not present

This weird signature is not supported by PHP which is why it's missing (having a param after variadics).